pkg/controller/topology.go

```diff
@@ -219,7 +216,7 @@ func aggregateTopologies(
 //
 // This flattening is then applied to all TopologySelectorTerms in AllowedTopologies, and
 // the resulting terms are OR'd together.
-func flatten(allowedTopologies []v1.TopologySelectorTerm) []topologyTerm {
+func flatten(topologyKeys []string, allowedTopologies []v1.TopologySelectorTerm) []topologyTerm {
```

Comment:
The `topologyKeys` parameter does not seem to be used inside `flatten()`. Can it be removed?

Reply:
sorry, came out of prep for https://github.com/kubernetes-csi/external-provisioner/pull/160

I can remove it and move the param into that PR if you prefer

Reply:
Ah I see. Moving it to #160 sounds great.